packages/components/src/components/modal/modal.jsx

```diff
@@ -29,7 +29,7 @@ const ModalElement = ({
     portalId,
     children,
     height,
-    should_header_stick_body,
+    should_header_stick_body = true,
```

Comment:
Does this affect any other usages of the modal?

Reply:
I checked them. They are ok.